packages/webdriverio/src/utils/interception/index.ts

```diff
@@ -25,8 +37,9 @@ export default class Interception {
             interval = this.browser.options.waitforInterval
         }
 
-        const fn = () => this.calls.length > 0
-        const timer = new Timer(interval, timeout, fn, true)
+        const fn = () => this.calls && this.calls.length > 0
+        // TODO: fix unknown type when Timer is typed
+        const timer = new Timer(interval, timeout, fn, true) as unknown as Promise<void>
```

Comment:
no need to add the todo given that timer hasn't been translated yet, also:
```suggestion
        // TODO: fix unknown type when Timer is typed
        const timer = new Timer(interval, timeout, fn, true) as unknown as Promise<boolean>
```

Reply:
Alright, removed

Reply:
Removed!